dobson/main.py

```diff
@@ -65,18 +67,38 @@ def respond_help(slack_client, channel):
     )
 
 
-def respond_users(slack_client, channel):
+def respond_users(slack_client, channel, list_all=False):
     """Respond with a list of users who are currently at ozone"""
     users = [device.user for device in utils.get_devices()]
+    unknown = list(utils.get_unknown_mac_addresses())
 
-    if len(users) > 2:
-        reply = '{}, and {} are at ozone'.format(', '.join(users[:-1]), users[-1])
-    elif len(users) == 2:
-        reply = '{} and {} are at ozone'.format(users[0], users[1])
+    # generic reply containing who is at ozone, e.g. "A, B and C are at ozone"
+    reply = list_to_str(users)
+    if len(users) == 0:
+        reply = 'Nobody is at ozone.'
     elif len(users) == 1:
-        reply = '{} is at ozone'.format(users[0])
+        reply += ' is at ozone.'
     else:
-        reply = 'Nobody is at ozone'
+        reply += ' are at ozone.'
+
+    if list_all and unknown:
+        # Add unknown Mac Addresses to the reply (if there are any)
+        unknown_string = list_to_str(unknown)
+        if users:
+            reply += ', along with the following untagged Mac Addresses: {}'.format(unknown_string)
+        else:
+            reply += '. But there are the following untagged Mac Addresses: '.format(unknown_string)
+    elif unknown:
+        # Add the number of unknown Mac Addresses to the reply (if there are any)
+        if users:
+            reply += ', along with {} unknown device{}.'.format(len(unknown), 's' if len(unknown) > 1 else '')
+        else:
+            reply += '. But there {} {} unknown device{}...'.\
+                format(
+                'are' if len(unknown) > 1 else 'is',
+                len(unknown),
+                's' if len(unknown) > 1 else ''
```

Comment:
This feels kinda like we need a pluralization class or something so we can just pass words into that and it can just return the proper plural, then it can handle all the cases. This is actually something I really like from Rails: https://apidock.com/rails/String/pluralize. Could also use https://pypi.org/project/inflect/, but not sure if we need a whole library for this.